source/_integrations/homekit.markdown

```diff
@@ -586,6 +586,8 @@ If you have any iOS 12.x devices signed into your iCloud account, media player e
 
 #### Accessories are all listed as not responding
 
+There are some reports where the IGMP settings of a router were causing issues. With the wrong IGMP settings all accessories will stop responding a few minutes after Home Assistant (re)started. To solve this, turn off your router IGMP settings: **IGMP Proxy**, **IGMP Snooping** and **IGMP Fast Leave**.
```

Comment:
This can have major consequences for other things. Like, for example, IPTV services, an general discoverability of other devices that rely on this. I'm not sure if this is a recommendation we should put up in the documentation to be honest.

Reply:
I agree, maybe it is better to rewrite it in an advice to check and play with the IGMP settings to see if there is an improvement and warn to double check if other things still work. For me changing the IGMP setting instantly solved my issues with HomeKit. Would that be something you could live with? I noticed more people in the community were helped by my post about the IGMP, so I thought maybe it can help more people from the documentation, because it took me literally 2 months to solve. 😄

Reply:
You hint it as a possible option, but clear that it might effect other things. Right now, it is a statement: Turn it off. We should be careful with that on these things. I guess...

Reply:
Agreed. Maybe this should be enough then. In my case i changed the default settings.

_There are reports where the IGMP settings in a router were causing issues with HomeKit. This resulted in a situation where all of the Home Assistant HomeKit accessories stopped responding a few minutes after Home Assistant (re)started. Double check your router's IGPM settings if you experiencing this issue. The default IGMP settings typically work best._

Reply:
I think that suggestion works well, lets do that!

Reply:
I've changed and committed the text.